rswag-specs/lib/rswag/specs/request_factory.rb

```diff
@@ -200,6 +200,10 @@ def build_json_payload(parameters, example)
       def doc_version(doc)
         doc[:openapi] || doc[:swagger] || '3'
       end
+
+      def extract_getter(parameter)
+        parameter[:getter]||parameter[:name]
```

Comment:
```suggestion
        parameter[:getter] || parameter[:name]
```
Suggestion for readability